src/components/ZarrMetadataTab.vue

```diff
@@ -0,0 +1,150 @@
+        <b-list-group-item v-for="array in group.arrays" :key="array.name">
+          <div class="d-flex justify-content-between array-name">
+            <h2>
+              <code>{{ array.name }}</code>
+            </h2>
+            <small>
+              <code>{{ array.dtype }}</code>
+            </small>
+          </div>
+          <div class="table-responsive array-dims">
+            <table class="table table-striped">
+              <thead>
+                <tr>
+                  <th>Dimension</th>
+                  <th>Shape</th>
+                  <th>Chunk Size</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr
+                  v-for="(dim, i) in array.dims"
+                  :key="array.name + '-' + dim"
+                >
+                  <td>
+                    <code>{{ dim }}</code>
+                  </td>
+                  <td>
+                    <code>{{ array.shape[i] }}</code>
+                  </td>
+                  <td>
+                    <code>{{ array.chunks[i] }}</code>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+          <div v-if="array.hasAttrs" class="mt-2 array-attrs">
+            <b-card no-body class="mb-1">
+              <b-card-header header-tag="header" class="p-1">
+                <b-button
+                  v-b-toggle="array.name + '-attrs'"
+                  block
+                  variant="light"
+                >
+                  <b>Attributes</b>
+                </b-button>
+              </b-card-header>
+              <b-collapse :id="array.name + '-attrs'">
+                <b-card-body>
+                  <dl>
+                    <template v-for="(value, key, i) in array.attrs">
+                      <dt :key="array.name + '-key-' + i">{{ key }}</dt>
+                      <dd :key="array.name + '-value-' + i">{{ value }}</dd>
+                    </template>
+                  </dl>
+                </b-card-body>
+              </b-collapse>
+            </b-card>
+          </div>
+        </b-list-group-item>
+      </b-list-group>
+      <div class="mt-2 zarr-attrs">
+        <b-card no-body class="mb-1">
+          <b-card-header header-tag="header" class="p-1">
+            <b-button v-b-toggle.zarr-attrs block variant="light">
+              <b>Group Attributes</b>
+            </b-button>
+          </b-card-header>
+          <b-collapse id="zarr-attrs">
+            <b-card-body>
+              <dl>
+                <template v-for="(value, key, i) in group.attrs">
+                  <dt :key="'group-key-' + i">{{ key }}</dt>
+                  <dd :key="'group-value-' + i">{{ value }}</dd>
+                </template>
+              </dl>
+            </b-card-body>
+          </b-collapse>
+        </b-card>
+      </div>
+    </div>
+  </b-tab>
+</template>
+
+<script>
+const axios = require("axios").default;
+
+function parseMeta(meta) {
```

Comment:
Why is this outside of the components and not inside the components methods?

Reply:
This was an oversight on my part - I'll move this to component methods.